codegen/smithy-aws-swift-codegen/src/main/kotlin/software/amazon/smithy/aws/swift/codegen/EndpointTestGenerator.kt

```diff
@@ -66,7 +66,7 @@ class EndpointTestGenerator(
                             val value = Value.fromNode(pair.second)
```

Comment:
Changes in this file fixes a bug in endpoint test generator where it was incorrectly & unnecessarily casting array endpoint params to `[AnyHashable]`. This was never caught before bc we never had test for it & no service has string array endpoint params yet.